### ISSUE TITLE
AS | Use hex encoding rather than base64 in parsed claims 

### ISSUE THREAD

**Issue by Xynnn007** [confidential-containers/trustee#372]
We are using hex in [tdx claims](https://github.com/confidential-containers/trustee/blob/main/attestation-service/verifier/src/tdx/claims.rs#L67, [sgx claims](https://github.com/confidential-containers/trustee/blob/main/attestation-service/verifier/src/sgx/claims.rs#L56), azure vTPM [claims](https://github.com/confidential-containers/trustee/blob/main/attestation-service/verifier/src/az_snp_vtpm/mod.rs#L76). Also base64 in [SNP](https://github.com/confidential-containers/trustee/blob/main/attestation-service/verifier/src/snp/mod.rs#L250).

I suggest that we should use a common encoding, which would make the policy writing work easier.

**Comment by larrydewey** [comment]
I agree, we should unify on a single encoding practice. If I remember correctly, I believe base64 was selected because of how the qemu interfaces were defined, though I could be remembering incorrectly. @fitzthum do you happen to know off the top of your head?

**Comment by Xynnn007** [comment]
Now SGX/TDX/SNP/SE/vTPM all are using hex.

**Comment by Xynnn007** [comment]
Looks like SNP is using base64 for parsed claims while hex for initdata

**Comment by Xynnn007** [comment]
> I agree, we should unify on a single encoding practice. If I remember correctly, I believe base64 was selected because of how the qemu interfaces were defined, though I could be remembering incorrectly. [@fitzthum](https://github.com/fitzthum) do you happen to know off the top of your head?

Qemu interface defines qom with [base64](https://github.com/confidential-containers/qemu/blob/amd-snp-202402240000/qapi/qom.json#L926-L929) for some launch configs. However, these configurations are not directly touched by users in the CoCo scenario
